src/app/api/job/[jobId]/route.ts

```diff
@@ -0,0 +1,51 @@
+export const dynamic = 'force-dynamic' // defaults to auto
+import { db } from '@/database'
+import { z } from 'zod'
+import { NextResponse } from 'next/server'
+import { wrapApiMemberAction } from '@/server/wrappers'
+import { requestingMember } from '@/server/context'
+
+const schema = z.object({
+    message: z.string().optional(),
+    // it's tempting to try to type this, but doesn't seem to work.
+    // not really needed though because the where clause below will error if an invalid status is present in the list below
+    status: z.enum(['RUNNING', 'ERRORED', 'RESULTS-REJECTED', 'RESULTS-APPROVED']),
```

Comment:
@nathanstitt - I think perhaps we are a bit out of sync in terms of the values here. The TOA is using [these](https://github.com/safeinsights/trusted-output-app/blob/b26d42e0aba4ddb35b0ae277e7eb267aa9e63fb1/src/app/api/run/%5BrunId%5D/route.ts#L4-L8) which I think are aligned with the latest version of the [brief](https://docs.google.com/document/d/18fZ6fYyFrsAznu9lhsTHge4VWuW0QLnl6DGkAD-kGRc/edit?tab=t.0) / [state diagram](https://docs.google.com/drawings/d/1UYeu8x5D1g2Pt6sJ3-lVtFJeY9ETOZYIl6177pXRga8/edit) :man_shrugging:

Reply:
hm, yes I ~may have~ did forgot to update this. thanks for noting it!

Reply:
just pushed update to correct this, thanks again